### PR TITLE
check_nt: change encoding from latin1 to utf8

### PR DESCRIPTION
**Pull request by RincewindsHat**

### CODE REVIEW

File: plugins/check_nt.c [hunk 1]
```diff
@@ -341,7 +341,7 @@ int main(int argc, char **argv){
 
 		2) If the counter you're going to measure is percent-based, the code will detect
 		 the percent sign in its name and will attribute minimum (0%) and maximum (100%)
-		 values automagically, as well the �%" sign to graph units.
+		 values automagically, as well the ¨%" sign to graph units.
```
Comment (datamuc) [inline plugins/check_nt.c:344]: U+000A8 doesn't make any sense to me here. I  just vote for U+00022 (")
```suggestion
		 values automagically, as well the "%" sign to graph units.
```

Comment (waja) [reply plugins/check_nt.c:344]: Looks reasonable...

**Comment by RincewindsHat** [comment]
done

**Comment by RincewindsHat** [comment]
@datamuc @waja ok?

**Comment by waja** [comment]
For me it looks reasonable.

**Comment by RincewindsHat** [comment]
@waja would you like to click on "approve" then? :-)

**Review by waja — APPROVED** [review]

**Comment by waja** [comment]
> @waja would you like to click on "approve" then? :-)

Always searching for the noob in the iOS app. ;)

**Pull request merged**